templates/private-sql-data-monitor.json

```diff
@@ -82,6 +82,13 @@
       "metadata": {
         "description": "How many seconds to wait between runs"
       }
+    },
+    "errorCountRows": {
+      "type": "int",
+      "defaultValue": 0,
+      "metadata": {
+        "description": "How many rows returned will trigger an error"
```

Comment:
technically it's how many rows is not error

Reply:
maybe something like "Greater than this many rows will be considered an error"

maybe something less wordy if you can think of it lol